test/e2e/runner/load.go

```diff
@@ -81,7 +81,7 @@ func loadGenerate(ctx context.Context, chTx chan<- types.Tx, multiplier int) {
 
 		select {
 		case chTx <- tx:
-			time.Sleep(time.Duration(100/multiplier) * time.Millisecond)
+			time.Sleep(time.Second / time.Duration(multiplier))
```

Comment:
why this change?

Reply:
I wanted to lower the rate of txs being sent to nodes - I think it's too high atm. I think I can revert this because it's not really relevant to this PR but I think it's what causes a lot of the timeouts